packages/ui/feature-builder-form-controls/src/lib/piece-properties-form/piece-properties-form.component.ts

```diff
@@ -389,63 +389,137 @@ export class PiecePropertiesFormComponent implements ControlValueAccessor {
       const validators: ValidatorFn[] = [];
       const prop = properties[pk];
       const propValue = propertiesValues[pk];
-      if (
-        prop.required &&
-        prop.type !== PropertyType.OBJECT &&
-        prop.type !== PropertyType.MARKDOWN &&
-        prop.type !== PropertyType.ARRAY
-      ) {
-        validators.push(Validators.required);
-      }
-      if (prop.type === PropertyType.OBJECT) {
-        controls[pk] = new UntypedFormControl(
-          propValue || prop.defaultValue || {},
-          validators
-        );
-      } else if (prop.type === PropertyType.ARRAY) {
-        controls[pk] = new UntypedFormControl(
-          propValue || prop.defaultValue || [''],
-          validators
-        );
-      } else if (prop.type === PropertyType.JSON) {
-        if (!this.customizedInputs || !this.customizedInputs[pk]) {
-          validators.push(jsonValidator);
+      switch (prop.type) {
+        case PropertyType.ARRAY: {
+          const controlValue = propValue
+            ? propValue
+            : Array.isArray(prop.defaultValue) && prop.defaultValue.length > 0
+            ? prop.defaultValue
+            : [''];
```

Comment:
Let's test wordpress, If i sent empty tags that would be problematic can you please doubel check?

I believe this should be null or empty